site/_docs/history.md

```diff
@@ -78,31 +166,420 @@ override logic to the `getMaxScale` and `getMaxPrecision` methods.
 #### New features
 {: #new-features-1-38-0}
 
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6603">CALCITE-6603</a>]
+  Lattice SQL supports generation of specified dialects
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6560">CALCITE-6560</a>]
+  Allow types with negative scale
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6576">CALCITE-6576</a>]
+  In `SET` clause of `UPDATE` statement, allow column identifiers to be prefixed
+  with table alias
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6584">CALCITE-6584</a>]
+  Validate prefixed column identifiers in `SET` clause of `UPDATE` statement
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4838">CALCITE-4838</a>]
+  Add `RoundingMode` in `RelDataTypeSystem` to specify the rounding behavior
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6020">CALCITE-6020</a>]
+  Add planner rule, `class ProjectOverSumToSum0Rule`, that converts `SUM` to
+  `SUM0`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6372">CALCITE-6372</a>]
+  Add `ASOF` join
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6519">CALCITE-6519</a>]
+  Support non-aggregate query that uses measure in `ORDER BY`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-4496">CALCITE-4496</a>]
+  Measure columns (`SELECT ... AS MEASURE`)
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5802">CALCITE-5802</a>]
+  In `RelBuilder`, add method `aggregateRex`, to allow aggregating complex
+  expressions such as "`1 + SUM(x + 2)`"
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6444">CALCITE-6444</a>]
+  Add a function library for Amazon Redshift
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6427">CALCITE-6427</a>]
+  Use a higher precision for `DECIMAL` intermediate results for some aggregate
+  functions like `STDDEV`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-5855">CALCITE-5855</a>]
+  Support frame exclusion in window functions, e.g.
+  `OVER (... EXCLUDE CURRENT ROW)`
+* [<a href="https://issues.apache.org/jira/browse/CALCITE-6365">CALCITE-6365</a>]
+  Support `RETURNING` clause of `JSON_QUERY` function
+
+New functions:
```

Comment:
nit: This section does not have the usual header highlighting and reference link; not sure if it is intentional.